src/cli/trn_cli_xdp.c

```diff
@@ -183,6 +183,62 @@ int trn_cli_load_transit_subcmd(CLIENT *clnt, int argc, char *argv[])
 	return 0;
 }
 
+int trn_cli_load_transit_offload_subcmd(CLIENT *clnt, int argc, char *argv[])
```

Comment:
Please take a look at this merged commit: https://github.com/CentaurusInfra/mizar/pull/445
I think you should be able to use existing function to load a program in offload mode instead of adding these new functions (which will require test )